frontend/js/components/CreateParty/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+import '../css/MainScreen.css';
+import { Button } from 'reactstrap';
+
+
+const CreateParty = () => {
```

Comment:
See below discussion for `JoinParty` for a more in-depth explanation, but should probably name this something like `SpotifyLogin`, and then have your `index.js` look like this:

```
export { default as CreatePartyName } from './CreatePartyName';
export { default as SpotifyLogin } from './SpotifyLogin';
```